nss-day-cohort-30/trestlebridge-farms-darling-ducks
# Pig model creation

# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change
Please delete options that are not relevant.

# Testing Instructions for Change Made
1. `git fetch --all`
1. `git checkout pig-model-creation`
1. Open VS Code and verify that Pig.cs is under Models/Animals directories and that no errors appear in the code. 

# Checklist:
